src/advanced_agent/monitoring/system_monitor.py

```
@@ -514,11 +519,59 @@ def get_latest_metrics(self) -> Optional[SystemMetrics]:
         """最新メトリクス取得"""
         return self.metrics_history[-1] if self.metrics_history else None
     
+    async def get_system_stats(self) -> Dict[str, Any]:
+        """システム統計情報取得（SelfLearningAgent用）"""
```

Ruff (0.12.2)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)